platform/darwin/src/NSDate+MGLAdditions.h

```diff
@@ -7,4 +7,7 @@
 /// Converts from a duration in seconds to a duration object usable in mbgl.
 mbgl::Duration MGLDurationInSeconds(NSTimeInterval duration);
 
+/// Converts from an mbgl duration object to a duration in seconds.
+NSTimeInterval MGLSecondsFromDuration(mbgl::Duration duration);
```

Comment:
`mbgl::Duration` is similar to `NSTimeInterval` in the sense that the unit is fixed and built into the type. You can't have one `mbgl::Duration` representing a duration "in seconds" and another representing a duration "in milliseconds". The actual unit is defined by `std::chrono::steady_clock` and I'm not sure that it's invariant across platforms or standard library implementations. But it is fixed at compile time.